### DISCUSSION TITLE
GitHub opens non-handled links in non-default browsers.

### DISCUSSION THREAD

**Discussion by elibroftw** [community/community#59713] [Mobile]
### Select Topic Area

Bug

### Body

I get an email from github from an issue. I want to unsubscribe. I click unsubcribe. Github app opens and immediately gives me a menu to open the link in a browser. Except the browsers available are everything but my default browser Firefox.

**Comment by aconsuegra** [comment]
Hi @elibroftw, thanks for reaching out with your issue. In which platform are you using GitHub Mobile?

**Reply by elibroftw** [reply]
Android
